phoenix-core/src/main/java/org/apache/phoenix/schema/stats/StatisticsWriter.java

```diff
@@ -74,6 +76,7 @@
  * Wrapper to access the statistics table SYSTEM.STATS using the HTable.
  */
 public class StatisticsWriter implements Closeable {
+    private static final Log LOG = LogFactory.getLog(StatisticsWriter.class);
```

Comment:
Same as above, use the same variable name as `LOGGER`

Reply:
Resolved